src/builder/commands/pip.rs

```diff
@@ -15,6 +18,10 @@ use builder::commands::generic::{run_command_at_env, capture_command};
 #[cfg(feature="containers")] use file_util::Dir;
 use build_step::{BuildStep, VersionError, StepError, Digest, Config, Guard};
 
+const PIP_HOME: &str = "/tmp/pip-install";
+const PYTHON_PATH: &str = concatcp!("/tmp/non-existent:", PIP_HOME);
```

Comment:
Why do we need `/tmp/non-existent` path here?

Reply:
This is some pip feature that tries to install something in the first directory in python path if that exists, or putting some `.pth` file there. I don't remember details unfortunately.

So if you try without it and rediscover the reason, please put the comment about that.

Reply:
Don't see anything strange with the current pip.

Reply:
Yes looks fine.